examples/contrib/detection/README.md

```diff
@@ -0,0 +1,30 @@
+# FasterRCNN Example with Ignite
+
+In this example, we show how to use _Ignite_ to train a image detection model with PyTorch built-in Faster RCNN implementation.
+
+- Using VOC2012 datasets (~2GB).
+- Vlidate with COCO mAP.
+- Visualization.
+- Aim as experiment manager.
+
+## Requirements:
+
+- pytorch-ignite: `pip install pytorch-ignite`
+- [torchvision](https://github.com/pytorch/vision): `pip install torchvision`
+- [aim](https://github.com/aimhubio/aim): `pip install aim`
```

Comment:
I think Albumentations and pycocotools deps are missing here and in requirements.txt